### DISCUSSION TITLE
Heirline blinks/disappears on entering insert mode

### DISCUSSION THREAD

**Discussion by olimorris** [rebelot/heirline.nvim#98] [Q&A] [answered]
![Screen Shot 2023-01-05 at 18 18 38](https://user-images.githubusercontent.com/9512444/210852234-6972c3ca-af0f-4350-90e8-e03dcfbd7b99.gif)

I've noticed that on entering insert mode, my status line disappears. It doesn't appear to relate to the `ViMode` component and wondered if anyone else has come across this?

**Comment by rebelot** [comment, marked as answer]
Have you tried with cmdheight != 0 ?  Also check if you have any insertenter autocmds that may produce some empty messages

**Reply by olimorris** [reply]
I actually shifted my `copilot` config away from `InsertEnter` and that seemed to do the trick
